modules/installing-ocp-agent.adoc

```diff
@@ -10,16 +10,29 @@ The following procedure deploys a single-node {product-title} in a disconnected
 
 .Procedure
 
-. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest[OpenShift mirror site] and download the latest version of the installer and client tarballs that match your operating system.
+. Log in to the {product-title} web console using your login credentials.
 
-. Extract the binary by running the following command:
+. Navigate to link: https://console.redhat.com/openshift/create/datacenter.
 +
-[source,terminal]
-----
-$ tar xvf <file> -C ~/.local/bin
-----
+image::agent_install.png[]
```

Comment:
Yes , its required .

Reply:
I agree with Steven that usually we try to avoid screenshots unless they are truly necessary, so most of our console procedures don't include them. Again, fine if you want them in this case, but something that we should try to avoid when possible.

Reply:
imo and in this context, it is required.